src/main/java/CreateCRE.java

```diff
@@ -0,0 +1,67 @@
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateCRE implements Job {
```

Comment:
Create CRE is not the best name I think - it suggest that responsibility of this class is creation of one CRE object which is not true. It should be rather something like 'CREResultsRetriever' or similar.